### ISSUE TITLE
BritBox showed then disappeared

### ISSUE THREAD

**Issue by yrp888** [Sandmann79/xbmc#693]
**Addon used**
- [X] Amazon VOD (plugin.video.amazon-test)
- [ ] Browser Launcher (plugin.program.browser.launcher)

- Addon version: 0.9.8

**Account type**
- [X] primevideo.com
- [ ] amazon.(com/co.uk/de/jp)

**System Setup (please provide the following information):**
- Hardware: NUC8 i5
- OS version: LibreElec 10.0.4
- Kodi version number: 19.5

**Upload Logs**

**Describe the bug**
Recently BritBox advertised a $2 for 2mths trial which was available on PrimeVideo so I signed up on 20 May. On our frequently used NUC after a while a "folder/category" eventually came up "BritBox $1 for 2months" (or words to that effect) and I added it to the Favourites.

She was able to browse all the BritBox categories and watch shows but last night (24 May) the Favourite stopped working with a Catalog Error. I removed the Favorite, cleared the Catalog cache, restarted Kodi but then would not show a BritBox category. I expected to see it in the "My Subscriptions" category but it's empty, I looked through all the categories/folders as well.

I cleared both caches and restarted, but BritBox is still not showing up anywhere.

I powered up the LG TV and went into PrimeVideo and of course I can see BritBox under My Subscriptions. I tried the other NUC8 on the LG and again, BritBox is nowhere to be seen.

Thankfully the various BritBox shows that had been watched are appearing in the "Continue Watching" category on both NUCs (and presumably on the RPi4) so that's a relief but means that we can't look at anything new.

**Comment by Sandmann79** [comment]
Please test the attached beta, this should solve the problem and show the channel under my subscriptions. 
Be sure to clear the catalogue cache after installation.
[plugin.video.amazon-test-0.9.9~beta+matrix.1.zip](https://github.com/Sandmann79/xbmc/files/11618117/plugin.video.amazon-test-0.9.9.beta%2Bmatrix.1.zip)

**Comment by yrp888** [comment]
Thank you for investigating @Sandmann79, I can confirm that 0.9.9-beta is now seeing the BritBox and other channels and was able to play a few that I tested.

Some series were missing episodes in most seasons so I did a Refresh on titles, and for the ones I tested it was OK, but for "Vera" it crashed on Season 12 on the RPi4. On the NUC8 I updated, Vera only goes to Season 11.

[kodi-20230601.zip](https://github.com/Sandmann79/xbmc/files/11619798/kodi-20230601.zip)

**Comment by yrp888** [comment]
I fired up the LG TV and its Prime app only showed 11 seasons for Vera. So for some reason the RPi4 thought there were 12 seasons which accounts for the error when refreshing a non-existent season, whereas the main NUC8 correctly shows 11 seasons.

On the NUC8 connected to the LG TV, after updating and clearing the Catalog, nothing other than my account name shows up in Prime. I cleared the cookies, both caches and logged in again but for some bizarre reason it still won't show anything.

The main NUC8 is fine, the RPi4 is packed away as we're about to go on a short break and it was OK earlier. If the RPi4 is behaving well when we arrive I might do another catalog only cache clear and report back.
